content/docs/introduction/faq.md

```diff
@@ -40,14 +40,10 @@ Python, and Ruby.
 
 ### How stable are Prometheus features, storage formats, and APIs?
 
-Although Prometheus and many of its ecosystem components are already quite
-stable, we will still allow for occasional breaking changes until the
-Prometheus server reaches version 1.0.0. These breaking changes will be pointed
-out in release announcements for components that already have a proper release
-process (like the Prometheus server) or communicated clearly otherwise. After
-releasing version 1.0.0, breaking changes will be indicated by increments of
-the major version. See also the documentation for [semantic
-versioning](http://semver.org/), which we are following.
+Within any given major version, we will only introduce fully backwards
+compatible changes. Any breaking changes will be indicated by increments
+of the major version. See also the documentation for
+[semantic versioning](http://semver.org/), which we are following.
```

Comment:
Valid point; given that this is just removing 1.0.0, I am not sure if keeping an up to date list there would make sense. Referring to a different place that lists them might make sense.